Core/ServeSync.Application/UseCases/EventManagement/Events/Commands/CreateEventCommandHandler.cs

```diff
@@ -24,21 +24,7 @@ public class CreateEventCommandHandler : ICommandHandler<CreateEventCommand, Gui
     private readonly IUnitOfWork _unitOfWork;
     private readonly ILogger<CreateEventCommandHandler> _logger;
     
-    public CreateEventCommandHandler(
-        IEventRepository eventRepository,
-        IBasicReadOnlyRepository<EventOrganization, Guid> eventOrganizationRepository,
-        IBasicReadOnlyRepository<EventOrganizationContact, Guid> eventOrganizationContactRepository,
-        IEventDomainService eventDomainService,
-        IUnitOfWork unitOfWork,
-        ILogger<CreateEventCommandHandler> logger)
-    {
-        _eventRepository = eventRepository;
-        _eventOrganizationRepository = eventOrganizationRepository;
-        _eventOrganizationContactRepository = eventOrganizationContactRepository;
-        _eventDomainService = eventDomainService;
-        _unitOfWork = unitOfWork;
-        _logger = logger;
-    }
+    
```

Comment:
Please rollback this change, you've just deleted code from create event command handler!